salesagility/SuiteCRM
# Feature/email addresses audit

## Description

New Module to audit email_addresses and email_addr_bean_rel  tables



## Motivation and Context

Record changes done on email_addresses and email_addr_bean_rel tables


## Types of changes


## How To Test This

-open an account/contact in edit mode
-edit the email addresses field by changing its value or any of the others option like invalid/op tout
-save the changes
-open the account/contact in detail view 
-click on the change log link next to the email field
-you should be able to see the change log

### Final checklist
